packages/reputation-miner/ReputationMiner.js

```diff
@@ -640,6 +645,7 @@ class ReputationMiner {
     } catch (err) { // eslint-disable-line no-empty
 
     }
+    gasEstimate = this.padGasEstimateIfNecessary(gasEstimate);
```

Comment:
The term `IfNecessary` here seems unecessarily vague -- why not something like `padGanacheEstimate`, as the only branching condition is whether the client is ganache or not.